1984bot.py

```diff
@@ -52,7 +52,79 @@
 @bot.event
 async def on_ready():
     print('ONLINE')
-
+    
+async def on_message(self, message):
+    if message.author == client.user:
+       return
+     string = message.content
+     stringArr = ['place', 'triggers', 'here']
+     #All below replaces characters in a string (common substitutions) to prevent people from escaping the blacklist
+     string = re.sub('1', 'i', string)
+     string = re.sub('3', 'e', string)
+     string = re.sub('4', 'a', string)
+     string = re.sub('5', 's', string)
+     string = re.sub('ñ', 'n', string)
+     string = re.sub('7', 't', string)
+     string = re.sub('0', 'o', string)
+     string = re.sub('8', 'b', string)
+     string = re.sub('&', 'and', string)
+     string = re.sub('z', 's', string)
+     string = re.sub('wanna', 'want to', string)
+     string = re.sub('your', '👇', string)
+     string = re.sub('ur', 'your', string)
+     string = re.sub('👇', 'your', string)

```

Comment:
What does this do? Isn't L73 unnecessary here?

Reply:
These 3 lines were originally created because "kill yourself" sometimes has "your" replaced with "ur" or pointing downwards.

Reply:
I see now, I was confused before cause I didn't realize that it was bcs `your` has `ur` in it lol. In the current version, these three lines have been replaced with a negative lookbehind.

Reply:
My reading comprehension do be kinda bad imma put line 75 back